components/NoteContent/HyperLinks.tsx

```diff
@@ -0,0 +1,33 @@
+import { MouseEvent } from "react";
+import { Anchor, Box } from "@mantine/core";
+import useStyles from "./NoteContent.styles";
+
+export const HyperLink = ({ href }: { href: string }) => {
+  const { classes } = useStyles();
+
+  if (/(gif|jpe?g|tiff?|png|webp|bmp)$/i.test(href)) {
+    return (
+      <Box sx={{ maxHeight: 200, overflow: "hidden" }}>
+        <img
+          src={href}
+          alt={href}
+          style={{ maxWidth: "50%", borderRadius: 12 }}
```

Comment:
make sure image has max height so we don't get god candled 😂

Reply:
the wrapper box crops it at max height 200. also i think i fixed the styling so the bottom corners are rounded.

Reply:
actually, i should make the max taller than that cause the image might be oriented the other way.

Reply:
ended up removing the container and setting the max height on the image to make sure the borders are always rounded.